exp3/tuning.py

```
@@ -0,0 +1,75 @@
+        for path in test_files[true_label]:
+            try:
+                test_feat = features.extract_mfcc(path)
+                min_dist = float('inf')
+                pred = None
+                for t_label, t_list in templates.items():
+                    for t_feat in t_list:
+                        d = dtw_core.calculate_dtw_distance(t_feat, test_feat)
+                        if d < min_dist:
+                            min_dist = d
+                            pred = t_label
+                if pred == true_label: correct += 1
```

Ruff (0.14.5)

Multiple statements on one line (colon)

(E701)